Octokit/Models/Response/ActivityPayloads/InstallationEventPayload.cs

```diff
@@ -4,8 +4,14 @@
 namespace Octokit
 {
     [DebuggerDisplay("{DebuggerDisplay,nq}")]
-    public class InstallationEventPayload : ActivityWithActionPayload
+    public class InstallationEventPayload
```

Comment:
Not inheriting from `ActivityWithActionPayload` will cause `GitHubService` not to be able to handle this event since we currently rely on it (see `DeserializeWebhookEvent`)

Reply:
As I told you IRL, these types aren't compatible anyway, so unfortunately it makes no sense for them to inherit from the same base class

Reply:
found a workaround